pkg/cmdrunner/cmdrunner.go

```diff
@@ -75,18 +82,30 @@ func (sr *ShellRunner) Run(options *RunOptions, format string, vars ...interface
 		}
 	}
 
-	// run
-	output, err := cmd.CombinedOutput()
-	if err != nil {
+	var stdOut, stdErr bytes.Buffer
+	var output []byte
+	var err error
+
+	cmd.Stdout = &stdOut
+	cmd.Stderr = &stdErr
+
+	if err = cmd.Run(); err != nil {
+		var exitCode int
 		sr.logger.DebugWith("Failed to execute command", "output", string(output), "err", err)
```

Comment:
Please also update these logs to the new command output (stdout/err) and get rid of the old unused variables

Reply:
Don't forget to also log the exit code, which effectively means this log entry should go below the next block